ang/crmUtil.js

```diff
@@ -19,7 +19,7 @@
       if (_.isObject(entity)) {
         // eval content is locally generated.
         /*jshint -W061 */
-        p = backend(eval('('+angular.toJson(entity)+')'), message);
+        p = backend(eval('('+angular.toJson(entity)+')'), action);
```

Comment:
Yea the CRM.api is overloaded with 2 signatures. And since js doesn't support "real" function overloading, we just have to document it well.